spec/client_spec.rb

```diff
@@ -124,52 +128,52 @@
     client.checkout.payments_api.payments_details(request_body)
   end
 
-  it "checks the creation of checkout url" do 
```

Comment:
I wonder if they're using a formatter that's intentionally adding a space here. I wouldn't know why, I don't see a rubocop file so it doesn't look like styling is being enforced anywhere.

Reply:
There's no formatter that I can find. I could change it back but it's something my editor settings do automatically: trim whitespace on save. Honestly, I can't imagine them caring one way or the other.